modular_skyrat/modules/modular_vending/code/clothesmate.dm

```diff
@@ -7,7 +7,6 @@
 		/obj/item/clothing/suit/toggle/jacket/flannel/red = 5,
 		/obj/item/clothing/suit/toggle/jacket/flannel/aqua = 5,
 		/obj/item/clothing/suit/toggle/jacket/flannel/brown = 5,
-		/obj/item/clothing/suit/hooded/wintercoat/custom = 5,
```

Comment:
What's this removed for?

Reply:
TG has one in the vendor; having a dupe item makes its total amount zero, and its better we remove our version instead of commenting out TG's version (which we did previously for.. some reason)